src/test/java/org/vaadin/artur/paperslider/TestViewIT.java

```diff
@@ -0,0 +1,16 @@
+package org.vaadin.artur.paperslider;
+
+import com.vaadin.flow.component.html.testbench.DivElement;
+import org.junit.Assert;
+import org.junit.Test;
+
+
+public class TestViewIT extends AbstractViewTest {
+
+    @Test
+    public void clickingButtonShowsNotification() {
```

Comment:
This method name does not align with what the test actually does.

Reply:
Done.